src/test/java/Homework18.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest {
+   @Test
+    public void playSong() throws InterruptedException {
+        navigateToPage();
```

Comment:
at this step `navigateToPage` inherits from `BaseTest `but in this function I see you are defile without value. So when initializing the driver and opening the browser, it will not open the koel website to perform the test